### PR TITLE
e2e tests

### PR DESCRIPTION
**Pull request by shaulzorea**
1. contains a single dummy test which uses “match method”
2. TODO: test has a hard coded string to alpha-lambda (need to either fetch from parameter store or to have vendor directory lambda itself use a static url)

### CODE REVIEW

File: packages/e2e/src/api/ApiFactory.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  }
+
+  create<Api extends BaseAPI>(ctor: (n: Configuration) => Api) : Api {
+    const configuration = new Configuration({
+      basePath: this.basePath,
+      fetchApi: fetch,
+    });
+
+    return ctor(configuration)
+      .withPreMiddleware(async (context) => {
```
Comment (shaikegross) [inline packages/e2e/src/api/ApiFactory.ts:20]: Can the preMiddleWare funciton be extracted. will be easier to understand what is being returned and why

File: packages/e2e/package.json [hunk 2]
```diff
@@ -0,0 +1,46 @@
+{
+  "name": "vendor-directory-api-e2e",
+  "version": "1.0.0",
+  "description": "end-to-end tests for vendor-directory-api",
+  "main": "out/index.js",
+  "scripts": {
+    "test": "NODE_ENV=test jest",
+    "lint": "eslint src --config .eslintrc",
+    "generate:models": "openapi-generator generate  -i ../vendor-directory-api/openapi.yaml -g typescript-fetch -o src/generated-clients/http-client-typescript --additional-properties=typescriptThreePlus=true",
+    "compile": "npx tsc",
```
Comment (shaikegross) [inline packages/e2e/package.json:10]: no need to write npx inside `scripts` block.

**Review by shaikegross — APPROVED** [review]

File: packages/e2e/package.json (hunk 2 above)
Comment (shaulzorea) [reply packages/e2e/package.json]: removed

File: packages/e2e/src/api/ApiFactory.ts (hunk 1 above)
Comment (shaulzorea) [reply packages/e2e/src/api/ApiFactory.ts]: extracted

**Pull request merged**